### ISSUE TITLE
Receiving 401 Unauthorized response when trying to redeem a used Nitro code

### ISSUE THREAD

**Issue by rikvik2006** [yafyz/nitrsnip#23]
I tested the application by sending Nitro links I had already used. 
However, I get attempts that have 401 Unauthorized as a response. 

I don't know what response I should expect using redeemed Nitro codes, however, I tried to redeem the same code by making a POST request to the endpoint https://discord.com/api/v9/entitlements/gift-codes/SP4Jmhyq2pJs676f/redeem defining the authorization header with an account token and I was receiving 404 as response saying that the code was unknown, which makes more sense as a response than 401.

![image](https://github.com/yafyz/nitrsnip/assets/75540124/1932b26b-cebb-43a0-9bd0-70e1d589569b)

In conclusion, is it normal to receive 401 when trying to redeem an already redeemed Nitro code?

**Comment by yafyz** [comment]
seems to work fine, you sure you didn't just make a typo in d_token?

**Comment by rikvik2006** [comment]
It was my fault for not defining `d_token`. I thought nitro was redeemed directly from tokens that found the nitro code.
My brain just skipped the line where the authorization header was defined in `gift.js`
